src/edit_mode/vi/mod.rs

```diff
@@ -50,6 +50,11 @@ impl EditMode for Vi {
                 (Mode::Insert, KeyModifiers::NONE, KeyCode::Char(c)) => {
                     ReedlineEvent::Edit(vec![EditCommand::InsertChar(c)])
                 }
+                (Mode::Insert, m, KeyCode::Char(c))
+                    if m == KeyModifiers::CONTROL | KeyModifiers::ALT =>
```

Comment:
What is this bitwise or of CONTROL and ALT here mean? Do you folks think it would be helpful if we add a comment explaining this? or a test case.

Reply:
I think I might have stupidly deleted this during a refactor. A comment or a test case would have stopped me from doing this :P

Reply:
Yeap, it was deleted and making non american keyboards not work correctly. The issue is that some keyboards need the special key `alt gr` which happens to be a combination of ALT and CTR modifiers. That's why it was there before